firstvoices/backend/search/utils/object_utils.py

```diff
@@ -79,25 +78,13 @@ def hydrate_objects(search_results, request):
 
     # Fetching objects from the database
     dictionary_objects = list(
-        DictionaryEntry.objects.filter(id__in=dictionary_search_results_ids)
-        .select_related("site", "created_by", "last_modified_by", "part_of_speech")
-        .prefetch_related(
-            "categories",
-            "acknowledgement_set",
+        DictionaryEntry.objects.filter(
+            id__in=dictionary_search_results_ids
+        ).prefetch_related(
             "translation_set",
-            "pronunciation_set",
-            "note_set",
-            "alternatespelling_set",
-            "site__language",
-            Prefetch(
-                "related_dictionary_entries",
-                queryset=DictionaryEntry.objects.visible(request.user)
-                .select_related("site")
-                .prefetch_related(
-                    "translation_set", *get_media_prefetch_list(request.user)
-                ),
-            ),
-            *get_media_prefetch_list(request.user),
+            "related_audio",
+            "related_images",
```

Comment:
Do we still need to prefetch the files to go with the media models?

Reply:
Yes, so taking this prefetching out causes about ~45 extra queries and the total response time increases by about ~1.5s

Reply:
Oh right. Nice Idea. I'll try it out and add it in if it helps in reducing some time.

Reply:
Interestingly, this helped in reducing the queries by a lot. 12 queries now instead of 35, but there is no noticeable response time reduction on local, but it can probably help on servers.